source/OptimaJet.DWKit.StarterApplication/Utility/DWKitHub.cs

```diff
@@ -0,0 +1,38 @@
+﻿using System;
```

Comment:
VSCode is being broken for me atm, so I don't have an easy way to clean this up :(